### PR TITLE
fix!: Use same HD Path as other testing environments

### PR DESCRIPTION
**Pull request by antazoey**
### What I did

It was a bit of mistake that we were not doing this and we never changed it... until now!
So now the test accounts are the same across the board, making it much easier to connect to nodes that Ape does not start itself.

### How I did it

Had the placeholder in the wrong spot.
Also pushing some fixes ape-foundry and ape-hardhat PRs.

### How to verify it

same accounts in 0.7

### Checklist

- [ ] All changes are completed
- [ ] New test cases have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: src/ape/utils/testing.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 DEFAULT_NUMBER_OF_TEST_ACCOUNTS = 10
 DEFAULT_TEST_MNEMONIC = "test test test test test test test test test test test junk"
-DEFAULT_HD_PATH = "m/44'/60'/0'/{}"
+DEFAULT_TEST_HD_PATH = "m/44'/60'/0'/0/"
```
[the remark this answers is not in the document]
Comment (antazoey) [reply src/ape/utils/testing.py]: Removing it does no harm and is better for eth-tester so went ahead with that.

File: tests/integration/cli/test_accounts.py [hunk 2]
```diff
@@ -171,17 +171,19 @@ def test_import_mnemonic_custom_hdpath(
...
     # export key
     result = runner.invoke(
         ape_cli,
         ["accounts", "export", ALIAS],
         input="\n".join([PASSWORD, PASSWORD]),
     )
     assert result.exit_code == 0, result.output
-    assert f"0x{PRIVATE_KEY}" in result.output
-    assert address in result.output
+    # NOTE: temp_keyfile uses the as the keyfile account.
```
[the remark this answers is not in the document]
Comment (antazoey) [reply tests/integration/cli/test_accounts.py]: [89af39b](https://github.com/ApeWorX/ape/pull/1774/commits/89af39b59893a0763120ba45e2b531b0e1abc6b9)

**Comment by fubuloubu** [comment]
Thank god!!!

**Comment by antazoey** [comment]
im gonna miss 0x1e59ce931B4CFea3fe4B875411e280e173cB7A9C

**Pull request merged**